LICENSE

```diff
@@ -17,3 +17,6 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
+
+All code created or modified by S�bastien Lucas <sebastien@slucas.fr> is licensed
+under GPL 2 (http://www.gnu.org/licenses/gpl.html).
```

Comment:
No problem, Shall I update the PR with the full [licence](https://spdx.org/licenses/MIT.html#licenseText) change